components/menu/src/auro-menu.js

```diff
@@ -212,10 +215,55 @@ export class AuroMenu extends LitElement {
       this.items.forEach((item) => {
         item.classList.remove('active');
         item.removeAttribute('selected');
+        item.removeAttribute('aria-selected');
       });
     }
   }
 
+  /**
+   * Toggle the selection state of the menuoption.
+   * @param {Object} option - The menuoption to toggle the selection state.
+   * @private
+   * @return {void}
+   */
+  toggleSelectionState(option) {
+    if (option.hasAttribute('selected')) {
+      this.handleDeselectState(option);
+    } else {
+      this.handleLocalSelectState(option);
```

Comment:
I am a little confused on the semantic naming of these functions. Why is one "Local" and the other isn't? Does that imply something?